manifests/minimal/manifest.yaml

```diff
@@ -1,6 +1,6 @@
 dist:
   module: github.com/observiq/bindplane-agent-ocb
-  name: observiq-distro
+  name: com.observiq.collector
```

Comment:
This seems pretty weird for the name of the collector, also not sure how it will work with windows where it adds a `.exe` to the end of the executable name. Is it possible to just set `service.name` in the supervisor config instead?